client/web/compose/src/views/Admin/Pages/Builder.vue

```diff
@@ -411,6 +419,23 @@ export default {
       this.page.blocks = this.blocks
     },
 
+    checkState ({ items, indicator }) {
```

Comment:
What are we doing here? This should only be checked on record pages, especially not on builder
Also not really sure what the point of this function is

Reply:
It caters to an edge case where a person tries to save a recordconfigurator with a non-configured condition. This method will also replace the one I am using in pageblock tabs to disable the save and close button